src/lib/components/PublishButton.js

```diff
@@ -63,7 +61,7 @@ export class PublishButtonComponent extends Component {
         >
           {(formik) => (
             <>
-              { ( formik.isSubmitting && formState === FORM_PUBLISHING ) ? (
```

Comment:
See comment above: this was made so that *only* this button is spinning when the form is publishing.